packages/block-editor/src/components/copy-handler/README.md

```diff
@@ -0,0 +1,42 @@
+# Copy Handler
+
+The `CopyHandler` component handles the copy, cut and paste events of its children blocks. It handles multiple selection of blocks as well.
+
+Concretely, it handles the display of success messages and takes care of copying the block to the clipboard using the [raw handling API](https://github.com/WordPress/gutenberg/tree/master/packages/blocks/src/api/raw-handling).
```

Comment:
>takes care of copying the block

Can you change this to `block/blocks`?

Reply:
You mean change "takes care of copying the block" to "takes care of copying the block/blocks"?